makefiles/boards/sam0.inc.mk

```diff
@@ -43,14 +43,8 @@ ifeq ($(PROGRAMMER),)
 endif
 
 ifeq ($(PROGRAMMER),edbg)
-  # use edbg for flashing
+  # use edbg for flashing. must be included before openocd
```

Comment:
Here I think that `openocd.inc.mk` could be included at the end of `edbg.inc.mk` instead of here. And then we could drop all the logic used here.
That would work because: FLASHER will defined first for edbg and DEBUGGER/DEBUGSERVER variables will be defined after by code in openocd.inc.mk. FLASHER won't be reset since it uses `?=` in `openocd.inc.mk`.

Reply:
Did that after testing on samr21-xpro, already pushed in a separate commit.